undertone/undertone-htb.js

```diff
@@ -244,12 +244,17 @@ function UndertoneHtb(configs) {
 
         var gdprValue = '';
         var consentString = '';
+        var uspString = '';
         var gdprConsent = ComplianceService.gdpr.getConsent();
         if (ComplianceService.isPrivacyEnabled() && gdprConsent) {
             gdprValue = gdprConsent.applies ? 1 : 0;
             consentString = gdprConsent.consentString;
         }
-        requestUrl += '&gdpr=' + gdprValue + '&gdprstr=' + consentString;
+
+        if (ComplianceService.isPrivacyEnabled() && ComplianceService.usp.getConsent()) {
+            uspString = ComplianceService.usp.getConsent();
```

Comment:
Hi,

We seem to have a bad url (http://hb.undertone.com/hb?pid=1&domain=null&gdpr=0&gdprstr=&ccpa=[object%20Object])
getConsent returns an object, the uspString should be ComplianceService.usp.getConsent().uspString

Thanks.

Reply:
Hi,

The HTTPS change looks good. Please make sure to fix the issue listed above.

Thanks

Reply:
Hello,

Just an FYI, by default the usp consent is an empty string.

Reply:
fixed that, please verify, thanks